server/clip_server/model/clip_trt.py

```diff
@@ -14,18 +14,19 @@
     )
 
 from clip_server.model.clip import _download, MODEL_SIZE
+from clip_server.model.clip_onnx import _MODELS as ONNX_MODELS
 
-_S3_BUCKET = 'https://clip-as-service.s3.us-east-2.amazonaws.com/models/tensorrt/'
-_MODELS = {
-    'RN50': ('RN50/textual.trt', 'RN50/visual.trt'),
-    'RN101': ('RN101/textual.trt', 'RN101/visual.trt'),
-    'RN50x4': ('RN50x4/textual.trt', 'RN50x4/visual.trt'),
-    # 'RN50x16': ('RN50x16/textual.trt', 'RN50x16/visual.trt'),
-    # 'RN50x64': ('RN50x64/textual.trt', 'RN50x64/visual.trt'),
-    'ViT-B/32': ('ViT-B-32/textual.trt', 'ViT-B-32/visual.trt'),
-    'ViT-B/16': ('ViT-B-16/textual.trt', 'ViT-B-16/visual.trt'),
-    'ViT-L/14': ('ViT-L-14/textual.trt', 'ViT-L-14/visual.trt'),
-}
+_MODELS = [
+    'RN50',
+    'RN101',
+    'RN50x4',
+    # 'RN50x16',
+    # 'RN50x64',
+    'ViT-B/32',
+    'ViT-B/16',
+    'ViT-L/14',
+    # 'ViT-L/14@336px',
```

Comment:
I reran some tests to reconvert the model, not only did it take ~30min to download and convert, but also met this error 
` [TRT] [E] 10: [optimizer.cpp::computeCosts::3628] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[onnx::MatMul_3417 + (Unnamed Layer* 87) [Shuffle]...MatMul_3011]}.)`
 Looks like it might be some problem with trt itself -> https://github.com/NVIDIA/TensorRT/issues/2124
 I would suggest temporarily removing the support of VL14 model for trt and wait for several new releases from trt.

Reply:
I'm fine to remove VL14 support. But make sure to 
- update the documentation
- raise an issue to track this problem.

Reply:
Please make sure to create a ticket to track this issue.